docs/source/io_formats/depletion_chain.rst

```diff
@@ -100,3 +100,11 @@ fissionable nuclides. It has the follow sub-elements:
 
       :data:
         Independent yields for each fission product
+
+In the event that a nuclide doesn't have any known fission product yields, it is
+possible to have that nuclide borrow yields from another nuclide by indicating
+the other nuclide in a single `parent` attribute. For example:
+
+.. code-block:: xml
+
+    <neutron_fission_yields parent="U235"/>
```

Comment:
I think I'll actually put this in the docstring for `Chain.from_endf` since it's specific to the logic there. It's certainly possible to create chains with other assumptions.